src/jets.cxx

```diff
@@ -66,11 +66,18 @@ namespace jet {
  * @param jes_shift JES shift variation (0 = nominal, +/-1 = up/down)
  * @param jer_shift JER shift variation ("nom", "up", or "down")
  * @param lhc_run LHC Run number, `2` for 2016-2018 and `3` for 2022-2026
+ * @param no_jer_for_unmatched_forward_jets if true, no jet energy resolution
+ * smearing is applied to jets in the forward region (|eta| > 2.5).
  * 
  * @return a dataframe with a new column of corrected jet \f$p_T\f$'s
  *
  * @note If jets with \f$p_T\f$ > 15 GeV are corrected, this change should be
  * propagated to the missing transverse momentum.
+ *
+ * @note The option `no_jer_for_unmatched_forward_jets` is introduced to
+ * mitigate jet horns appearing in the eta distribution of jets at
+ * \f$|\eta| \sim 2-3\f$ in Run 3 analyses. This option should only set to
+ * `true` if the recipe to mitigate jet horns is implemented in the analysis.
```

Comment:
I would also add here that they should not be matched to a gen. jet as you also describe in the PR.